devel/mxml/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        michaelrsweet mxml 3.1 v
+
+categories          devel textproc
+platforms           darwin
+license             Apache-2.0
```

Comment:
This should be:
```
license             Apache-2
```

Reply:
Both Apache-2 and Appache-2.0 was in the list of licenses in: `date && port -q info --license --index all | grep -v ^-- | tr ... ... ...`. So, I checked with the list at [SPDX](https://spdx.org/licenses/) to see, and they only used the 2.0 version.